polaris/polaris/management/commands/process_pending_deposits.py

```diff
@@ -441,27 +575,56 @@ async def get_or_create_destination_account(
-                    raise RuntimeError(
-                        "an exception was raised while attempting to create the destination "
-                        f"account for transaction {transaction.id}"
-                    )
-                finally:
-                    if (
-                        distribution_account in locks["source_accounts"]
-                        and locks["source_accounts"][distribution_account].locked()
-                    ):
-                        logger.debug(
-                            f"unlocking after creating destination accoutn for transaction {transaction.id}"
+                attempt = 1
+                transaction_hash = None
+                while True:
+                    try:
+                        logger.info(
+                            f"calling create_destination_account, attempt #{attempt}"
+                        )
+                        transaction_hash = await sync_to_async(
+                            rci.create_destination_account
+                        )(transaction=transaction)
+                    except (
+                        TransactionSubmissionBlocked,
+                        TransactionSubmissionFailed,
+                        TransactionSubmissionPending,
+                    ) as e:
+                        await sync_to_async(
+                            cls.handle_transaction_submission_exception
+                        )(transaction, e, str(e))
+                        if type(e) == TransactionSubmissionPending:
+                            logger.info(
+                                f"TransactionSubmissionPending raised by create_destination_account(), "
+                                f"re-submitting transaction {transaction.id}"
+                            )
+                            attempt += 1
+                            continue
+                    except Exception:
+                        raise RuntimeError(
+                            "an exception was raised while attempting to create the destination "
+                            f"account for transaction {transaction.id}"
                         )
-                        locks["source_accounts"][distribution_account].release()
+                    finally:
+                        if (
+                            distribution_account in locks["source_accounts"]
+                            and locks["source_accounts"][distribution_account].locked()
+                        ):
+                            logger.debug(
+                                f"unlocking after creating destination account for transaction {transaction.id}"
+                            )
+                            locks["source_accounts"][distribution_account].release()
+                    break
+
+                if transaction_hash:  # accont was created
+                    transaction.submission_status = Transaction.SUBMISSION_STATUS.ready
+                    transaction.status_message = ""
```

Comment:
I think we can set this to `None` since the column is nullable. I see a couple instances of this.